root/edit/details/EditAlias.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+
+import React from 'react';
+import _ from 'lodash';
+
+import formatEntityTypeName from '../../static/scripts/common/utility/formatEntityTypeName';
+import bracketed from '../../static/scripts/common/utility/bracketed';
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import isolateText from '../../static/scripts/common/utility/isolateText';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import locales from '../../static/scripts/common/constants/locales';
+import FullChangeDiff from '../../static/scripts/edit/components/edit/FullChangeDiff';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type CompT<T> = {
+  new: T,
+  old: T,
+};
+
+type OnlyNameT = {
+  name: string,
+} | null;
+
+type DisplayDataT = {
+  alias: CompT<string>,
+  artist?: OnlyNameT,
+  begin_date: CompT<PartialDateT>,
+  end_date: CompT<PartialDateT>,
+  ended: CompT<boolean>,
+  entity_type: string,
+  locale: CompT<string | null>,
+  primary_for_locale: CompT<boolean>,
+  sort_name: CompT<string>,
+  type: {
+    new: OnlyNameT,
+    old: OnlyNameT,
+  },
+};
+
+type EditAliasT = {
+  locale: string,
+  name: string,
+  primary_for_locale: boolean,
+};
+
+type Props = {
+  alias: EditAliasT,
+  display_data: DisplayDataT,
+  edit_kind: string,
+};
+
+const EditAlias = ({edit}: {edit: Props}) => {
+  const display = edit.display_data;
+  const entityType = display.entity_type;
+  const entity = display[entityType];
+  const aliasName = edit.alias.name;
+  const changesExist = display.alias.new === display.alias.old &&
+    display.sort_name.new === display.sort_name.old &&
+    display.locale.new === display.locale.old &&
+    _.isEqual(display.type.new, display.type.old) &&
+    _.isEqual(display.begin_date.new, display.begin_date.old) &&
+    _.isEqual(display.end_date.new, display.end_date.old) &&
+    display.ended.new === display.ended.old;
+  return (
+    <>
+      {changesExist ? (
+        <p>{l('There are no changes')}</p>
```

Comment:
I'll rename the variable. Thanks :)

Reply:
This doesn't exist in the original template though, so should be removed.

If this is something you need to show in the edit form, it should be moved there - and there I'd expect "There are no changes" to be displayed only once, not for each possible edit type. This isn't useful in the general edit display because edits being displayed should always have changes.

Reply:
okay, I'm put the code in the forms :)